src/base/UIni.pas

```diff
@@ -503,13 +506,13 @@   TIni = class
 
   IPreviewFadingTranslated:    array[0..5] of UTF8String  = ('Off', '1 Sec', '2 Secs', '3 Secs', '4 Secs', '5 Secs');
 
-  ILyricsFontTranslated:       array[0..2] of UTF8String = ('Plain', 'OLine1', 'OLine2');
+  ILyricsStyleTranslated:      array[0..2] of UTF8String = ('Regular', 'Bold', 'Outline');
```

Comment:
- [ ] we need some compatibility for old config.ini files regarding the ILyricsStyleTranslated